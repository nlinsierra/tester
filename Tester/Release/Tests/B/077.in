9S 2D 3S 5D 6H 3H 5C 9C 6D 7H KH QD 2H 8H JD 5H 3C QH KS 6C TS JS JC 6S TH KD 7D 7C AC
