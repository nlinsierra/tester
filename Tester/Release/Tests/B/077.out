23
2C 4C 8C TC QC KC 3D 4D 8D 9D TD AD 4H 9H JH AH 2S 4S 5S 7S 8S QS AS
